.github/stale.yml

```diff
@@ -0,0 +1,55 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels: []
+#  - pinned
```

Comment:
Should we consider exempting Issue-Meta and Issue-Discussion?

Reply:
and WIP PRs?

Reply:
We will enable the bot only for PRs for now, so `Issue-Meta` and `Issue-Discussion` don't apply.
For `WIP` PRs, I think we should treat them the same way.

Reply:
the issue labels are for issues, not PRs.  I don't see any reason to exempt them.